test/conformance/tests/test_sriov_operator.go

```diff
@@ -306,6 +306,29 @@ var _ = Describe("[sriov] operator", func() {
 				g.Expect(newLease.Spec.HolderIdentity).ToNot(Equal(oldLease.Spec.HolderIdentity))
 			}, 30*time.Second, 5*time.Second).Should(Succeed())
 		})
+
+		Context("SriovNetworkMetricsExporter", func() {
+			It("should be deployed if the feature gate is enabled", func() {
```

Comment:
for a following up PR please add another test to
1. create a pod with sriov device
2. send some traffic
3. access the metrics pod on that node run curl and check that we have the metrics
4. for OCP only do a curl to the Prometheus and check that we are able to pull the info

be aware we need to skip virtual cluster for the test as the IGB driver doesn't support metrics

Reply:
yes! I'm preparing an e2e test in:
- https://github.com/k8snetworkplumbingwg/sriov-network-operator/pull/702

I will add a prometheus test there